Automattic/wp-calypso
# Invites: move /accept-invite to /my-sites/invites

### How to test ###

Everything in the invite process should still flow.

* checkout `update/move-accept-invite-to-invites`
* invite yourself
* go to `http://calypso.dev:3000/accept-invite/<blog_id>/<invitation_id>`

Closes #1101
